src/main/java/net/sf/jabref/logic/fileformats/medline/AccessionNumberList.java

```diff
@@ -0,0 +1,78 @@
+//
+// This file was generated by the JavaTM Architecture for XML Binding(JAXB) Reference Implementation, v2.2.8-b130911.1802 
+// See <a href="http://java.sun.com/xml/jaxb">http://java.sun.com/xml/jaxb</a> 
+// Any modifications to this file will be lost upon recompilation of the source schema. 
+// Generated on: 2016.06.26 at 02:28:06 PM CEST 
+//
+
+
+package net.sf.jabref.logic.fileformats.medline;
+
+import java.util.ArrayList;
+import java.util.List;
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+import javax.xml.bind.annotation.XmlType;
+
+
+/**
+ * <p>Java class for anonymous complex type.
+ * 
+ * <p>The following schema fragment specifies the expected content contained within this class.
+ * 
+ * <pre>
+ * &lt;complexType>
+ *   &lt;complexContent>
+ *     &lt;restriction base="{http://www.w3.org/2001/XMLSchema}anyType">
+ *       &lt;sequence>
+ *         &lt;element ref="{}AccessionNumber" maxOccurs="unbounded"/>
+ *       &lt;/sequence>
+ *     &lt;/restriction>
+ *   &lt;/complexContent>
+ * &lt;/complexType>
+ * </pre>
+ * 
+ * 
+ */
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlType(name = "", propOrder = {
+    "accessionNumber"
+})
+@XmlRootElement(name = "AccessionNumberList")
+public class AccessionNumberList {
+
+    @XmlElement(name = "AccessionNumber", required = true)
+    protected List<String> accessionNumber;
```

Comment:
I would say rename it to `accessionNumbers` or `accessionNumberList` or `accessionNumbersList`. But since this applies to all auto generated files I don't know if it makes much sense to change it in all of the files.